tests/e2e/fixtures/statsFixtures.js

```diff
@@ -263,6 +262,64 @@ const seasonTwoMatches = [
   },
 ];
 
+const seasonFourMatches = [
+  ////////////
+  // Week 1 //
+  ////////////
+  // Player 1 week 1
+  {
+    player1: 'player1',
+    player2: 'player2',
+    winner: 'player2',
+    startTime: dayjs().subtract(1, 'day').valueOf(),
+    endTime: dayjs().subtract(1, 'day').valueOf(),
+  },
+  {
+    player1: 'player1',
+    player2: 'player3',
+    winner: 'player1',
+    startTime: dayjs().subtract(2, 'day').valueOf(),
+    endTime: dayjs().subtract(2, 'day').valueOf(),
```

Comment:
Similarly to be in week 1 of a season that started 2 weeks ago, these need to be 2 weeks behind
```suggestion
    startTime: dayjs().subtract(2, 'week').add(2, 'day').valueOf(),
    endTime: dayjs().subtract(2, 'week').add(2, 'day').valueOf(),
```